README.md

```diff
@@ -248,44 +159,26 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project star the repo
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+We would like to thank the Microverse community for giving me the opportunity of learning with them.
+I would like like to extend more thanks to my code reviewer who have been there for me guiding in the day today activies to improve my technical skills in development
+More thanks goes to my coding partner, morning sessions and standup teams who have been getting me out of  blockages when i needed help and guidance. 
 
```

Comment:
- [ ] The Creative Commons license of the design requires you to give appropriate credit to the author. ✍️ Please include this in your project's README as a way to acknowledge the original designer. 🙏🎨

Reply:
- [ ] @mumbereh Great work implementing the changes requested however you forgot to add an acknowledgment that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). as previously requested, this is part of the project requirement
![image](https://github.com/mumbereh/react_redux_capstone/assets/61408860/121606c7-0ee2-4bae-bdd2-f27ff5934bbe)